maintenance/passport-azure-ad/lib/jwe.js

```diff
@@ -507,17 +507,34 @@ exports.decrypt = (jweString, jweKeyStore, log, callback) => {
      *  cek decryption
      ***************************************************************************/
     var cek_result = decryptCEK(header, encrypted_cek, jweKeyStore, log);
-    if (cek_result.error)
+    if (cek_result.error) {
+      if (cek_result.error === "converting jwk to RSA privatePemKey to decrypt cek") {
+        jose.JWK.asKey(cek_result.cek, "pem").then((pemKey) => {
```

Comment:
I'm not sure what the expected support for es6 syntax is, but might want to have this be a non-arrow function just to be safe.

Reply:
We use arrow functions in multiple places in passport today, is there a specific concern with the arrow function here?

Reply:
Nope, just wanted to be sure given this lib doesnt use Typescript or any other transpiling.